scripts/governance/RevokeMultiSig.s.sol

```diff
@@ -0,0 +1,254 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.20;
+
+import { console } from "forge-std/console.sol";
+import { IVaultManagerFunctions } from "borrow/interfaces/IVaultManager.sol";
+import { IAgToken } from "borrow/interfaces/IAgToken.sol";
+import { IERC721Metadata } from "oz/token/ERC721/extensions/IERC721Metadata.sol";
+import { IAccessControl } from "oz/access/IAccessControl.sol";
+
+import { Utils } from "../Utils.s.sol";
+import "../Constants.s.sol";
+
+interface ITreasuryWithRole {
+    function addMinter(address minter) external;
+
+    function removeMinter(address minter) external;
+}
+
+contract RevokeMultiSig is Utils {
```

Comment:
This exact one is useless as it is specific to introduce the on chain governance